neqo-transport/src/connection/tests/recovery.rs

```diff
@@ -255,43 +256,51 @@ fn pto_handshake_complete() {
     assert_eq!(*client.state(), State::Connected);
 
     let cb = client.process_output(now).callback();
-    assert_eq!(cb, HALF_RTT * 6);
+    assert_eq!(cb, pto);
 
     let mut pto_counts = [0; MAX_PTO_COUNTS];
     assert_eq!(client.stats.borrow().pto_counts, pto_counts);
 
     // Wait for PTO to expire and resend a handshake packet.
     // Wait long enough that the 1-RTT PTO also fires.
     qdebug!("---- client: PTO");
-    now += HALF_RTT * 6;
+    now += pto;
     let pkt2 = client.process_output(now).dgram();
     assert_handshake(pkt2.as_ref().unwrap());
 
     pto_counts[0] = 1;
     assert_eq!(client.stats.borrow().pto_counts, pto_counts);
 
+    // PTO has been doubled.
+    let pto = 2 * pto;
+    let cb = client.process_output(now).callback();
+    assert_eq!(cb, pto);
+
     // Get a second PTO packet.
     // Add some application data to this datagram, then split the 1-RTT off.
     // We'll use that packet to force the server to acknowledge 1-RTT.
     let stream_id = client.stream_create(StreamType::UniDi).unwrap();
     client.stream_close_send(stream_id).unwrap();
-    now += HALF_RTT * 6;
+    now += pto;
     let pkt3 = client.process_output(now).dgram();
     assert_handshake(pkt3.as_ref().unwrap());
     let (pkt3_hs, pkt3_1rtt) = split_datagram(&pkt3.unwrap());
     assert_handshake(&pkt3_hs);
     assert!(pkt3_1rtt.is_some());
 
     // PTO has been doubled.
+    let pto = pto * 2;
     let cb = client.process_output(now).callback();
-    assert_eq!(cb, HALF_RTT * 12);
+    assert_eq!(cb, pto);
 
-    // We still have only a single PTO
+    // Two PTOs in a row.
+    pto_counts[0] = 0;
+    pto_counts[1] = 1;
     assert_eq!(client.stats.borrow().pto_counts, pto_counts);
```

Comment:
See documentation for `Stats::pto_counts`:

https://github.com/mozilla/neqo/blob/3b8fac0f33b2a36337ed36fa6cc8d7e7b01ca42b/neqo-transport/src/stats.rs#L184-L186